libs/api/v1/src/lib/api-v1.ts

```diff
@@ -194,3 +198,4 @@ export const apiV1 = new Elysia({ prefix: '/v1' })
       },
     ),
   )
+  .group('/dev', (devApp) => devApp.get('/test', 'Hello World'))
```

Comment:
**suggestion:** (non-blocking) testing API would be great to have in new package (in `libs/api/dev`) so that it clearly indicates the purpose. V1 is something that should be seen as stable. Especially after we publish the API and it's used, we shouldn't do any breaking changes.

Reply:
migrated